lib/cubit/payment_limits/payment_limits_cubit.dart

```diff
@@ -0,0 +1,70 @@
+library payment_limits_cubit;
+
+import 'package:breez_sdk_liquid/breez_sdk_liquid.dart';
+import 'package:breez_translations/breez_translations_locales.dart';
+import 'package:flutter_breez_liquid/flutter_breez_liquid.dart';
+import 'package:flutter_fgbg/flutter_fgbg.dart';
+import 'package:hydrated_bloc/hydrated_bloc.dart';
+import 'package:l_breez/cubit/payment_limits/payment_limits_state.dart';
+import 'package:l_breez/utils/exceptions.dart';
+import 'package:logging/logging.dart';
+
+export 'payment_limits_state.dart';
+
+class PaymentLimitsCubit extends Cubit<PaymentLimitsState> {
```

Comment:
I think we should override here the onClose functino and close the fgbgevents stream.
This bloc is created on each page creation and the bloc framework calls the close function when the page is disposed.

Reply:
Done.
[Close foreground/background event notifier stream on close()](https://github.com/breez/l-breez/pull/63/commits/45b424eec85be97797f6d8e81bc944f7fc1decc1)